lib/beaker-pe/install/pe_utils.rb

```diff
@@ -241,7 +242,7 @@ def installer_cmd(host, opts)
           else
             pe_debug = host[:pe_debug] || opts[:pe_debug]  ? ' -D' : ''
             pe_cmd = "cd #{host['working_dir']}/#{host['dist']} && ./#{host['pe_installer']}#{pe_debug}"
-            if ! version_is_less(host['pe_ver'], '2016.2.1')
+            if ! version_is_less(host['pe_ver'], '2016.2.1') && ! opts[:interactive]
```

Comment:
as long as no one gets confused and passes this in as a string, I think this is good to go

Reply:
```
   ! nil
=> true
   ! false
=> true
   ! true
=> false
```
^ Double checked that I've not forgotten how Ruby works.